docs/developer_guide/schemas.rst

```diff
@@ -0,0 +1,374 @@
+                  "type": "string"
+                },
+                "resolution": {
+                  "description": "The smallest meaningful difference (in specified unit) between values in data",
+                  "type": "number",
+                  "default": -1
+                },
+                "conversion": {
+                  "description": "Scalar to multiply each element in data to convert it to the specified unit",
+                  "type": "number",
+                  "default": 1
+                },
+                "starting_time": {
+                  "description": "The timestamp of the first sample",
+                  "type": "number"
+                },
+                "rate": {
+                  "description": "Sampling rate in Hz",
+                  "type": "number"
+                },
+                "comments": {
+                  "description": "Human-readable comments about this TimeSeries dataset",
+                  "type": "string",
+                  "default": "no comments"
+                },
+                "description": {
+                  "description": "Description of this TimeSeries dataset",
+                  "type": "string",
+                  "default": "no description"
+                },
+                "control": {
+                  "description": "Numerical labels that apply to each element in data",
+                  "type": "array"
+                },
+                "control_description": {
+                  "description": "Description of each control value",
+                  "type": "array"
+                },
+                "offset": {
+                  "description": "Scalar to add to each element in the data scaled by 'conversion' to finish converting it to the specified unit.",
+                  "type": "number",
+                  "default": 0
+                }
+              },
+              "type": "object",
+              "additionalProperties": false,
+              "tag": "pynwb.ecephys.ElectricalSeries"
+            }
+          },
+          "type": "object",
+          "additionalProperties": false,
+          "tag": "Ecephys"
+        }
+      },
+      "type": "object",
+      "additionalProperties": false,
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "$id": "metadata.schema.json",
+      "title": "Metadata",
+      "description": "Schema for the metadata",
+      "version": "0.1.0"
+    }
+
+Like with the source schema, metadata schema are combined automatically when combined in an ``NWBConverter``, which
+can also be accessed using ``.get_metadata_schema``, though the underlying schemas are merged directly with each other
+instead of being combined on a key basis.
```

Comment:
Hmm... This point isn't super clear, but if your intent isn't to get too deep into the distinction, at a minimum I'd include something like 'For more information on how these nested dictionaries automatically merge, refer to https://neuroconv--256.org.readthedocs.build/en/256/api/utils.html#neuroconv.utils.dict.dict_deep_update' (with a better sphinx link to that API doc, of course)